python/tvm/relay/backend/vm.py

```diff
@@ -253,6 +254,12 @@ def _make_executor(self, expr=None):
 
         def _vm_wrapper(*args, **kwargs):
             args = self._convert_args(main, args, kwargs)
+            ret_type = self.mod["main"].checked_type.ret_type
+            if type_has_any(ret_type) and "llvm" not in str(self.target) and "arm" not in str(
+                    self.target):
+                raise ValueError(
+                    "Virtual Machine only supports static graphs on CPU, got output type",
```

Comment:
Should be "Virtual Machine only supports static graphs on non-CPU platforms"?

Reply:
Thanks for catching it!